samples/device_ping/src/device_ping_lin.json

```diff
@@ -0,0 +1,29 @@
+{
+    "modules": [
+        {
+            "module name": "logger_hl",
```

Comment:
Because we got rid of HL modules, references like this should be changed to something like:
```
    {
        "module name": "logger",
        "module path": "./modules/logger/liblogger.so",
        "args": {
            "filename": "log.txt"
        }
    },
```